v4l2/v4l2_wrapper.cpp

```diff
@@ -206,3 +207,66 @@ int32_t YamiV4L2_SetDrmFd(int32_t fd, int drm_fd)
      return ret;
 }
 #endif
+
+int32_t YamiV4L2_SetParameter(int32_t fd, const char* key, const char* value)
+{
+    int ret = -1;
+
+    if (!key || !value) {
+        ERROR("invalue parameter\n");
+        return -1;
+    }
+
+    V4l2CodecPtr v4l2Codec = _findCodecFromFd(fd);
+    ASSERT(v4l2Codec);
+
+    if (!strcmp(key, "frame-memory-type")) {
```

Comment:
Do we really need so much output type for v4l2. Could we just keep one for real usage?